<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Plot_Definer</class>
 <widget class="QWidget" name="Plot_Definer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1" colspan="3">
    <widget class="Variable_Box" name="lineEdit_x_axis"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>x-axis:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="3" rowspan="4">
    <widget class="QLabel" name="fit_definer">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>No y-axis selected</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QCheckBox" name="checkBox_same_fit">
     <property name="text">
      <string>same fit for all </string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QWidget" name="y_axes" native="true"/>
   </item>
   <item row="4" column="2" rowspan="5">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0" rowspan="5" colspan="2">
    <widget class="QWidget" name="plotting_group" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="4" column="3">
       <widget class="Variable_Box" name="lineEdit_ylabel2"/>
      </item>
      <item row="3" column="0" colspan="4">
       <widget class="QCheckBox" name="checkBox_xlog">
        <property name="text">
         <string>x logarithmic?</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string># points:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>y-label:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_top_left_x">
        <property name="toolTip">
         <string>Set the exact x coordinate of the top left corner of the plot window in pixels (min 0)</string>
        </property>
        <property name="text">
         <string>Top Left X</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="lineEdit_top_left_x">
        <property name="toolTip">
         <string>Minimum is 0 pixels</string>
        </property>
       </widget>
      </item>
      <item row="9" column="3">
       <widget class="QLineEdit" name="lineEdit_plot_width">
        <property name="toolTip">
         <string>Minimum is 430 pixels</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_top_left_y">
        <property name="toolTip">
         <string>Set the exact y coordinate of the top left corner of the plot window in pixels (min 0)</string>
        </property>
        <property name="text">
         <string>Top  Left Y</string>
        </property>
       </widget>
      </item>
      <item row="10" column="3">
       <widget class="QLineEdit" name="lineEdit_plot_height">
        <property name="toolTip">
         <string>Minimum is 126 pixels</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBox_ylog">
        <property name="text">
         <string>y logarithmic?</string>
        </property>
       </widget>
      </item>
      <item row="10" column="2">
       <widget class="QLabel" name="label_plot_height">
        <property name="toolTip">
         <string>Set the height of the plot window in pixels (min 126)</string>
        </property>
        <property name="text">
         <string>Plot Height</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QLineEdit" name="lineEdit_nPoints"/>
      </item>
      <item row="7" column="0" colspan="4">
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="3">
       <widget class="Variable_Box" name="lineEdit_xlabel"/>
      </item>
      <item row="1" column="1" colspan="3">
       <widget class="Variable_Box" name="lineEdit_title"/>
      </item>
      <item row="4" column="1">
       <widget class="Variable_Box" name="lineEdit_ylabel"/>
      </item>
      <item row="10" column="1">
       <widget class="QLineEdit" name="lineEdit_top_left_y">
        <property name="toolTip">
         <string>Minimum is 0 pixels</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>x-label:</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QCheckBox" name="checkBox_show_in_browser">
        <property name="toolTip">
         <string>Check this to make the plot available via the browser.
This will significantly increase the protocol startup time!
Reachable under 127.0.0.1:Port</string>
        </property>
        <property name="text">
         <string>Show plot in browser</string>
        </property>
       </widget>
      </item>
      <item row="9" column="2">
       <widget class="QLabel" name="label_plot_width">
        <property name="toolTip">
         <string>Set the width of the plot window in pixels (min 430)</string>
        </property>
        <property name="text">
         <string>Plot Width</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2" colspan="2">
       <widget class="QCheckBox" name="checkBox_ylog2">
        <property name="text">
         <string>y2 logarithmic?</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>title:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>y-label 2</string>
        </property>
       </widget>
      </item>
      <item row="13" column="0" colspan="4">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="8" column="0">
       <widget class="QCheckBox" name="checkBox_manual_plot_position">
        <property name="toolTip">
         <string>Check this to set the manual x&amp;y coordinates and the size of the plot window.</string>
        </property>
        <property name="text">
         <string>Manual plot position</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label_port">
        <property name="toolTip">
         <string>Port of the localhost to display the plot</string>
        </property>
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QSpinBox" name="spinBox_port">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
        </property>
        <property name="maximum">
         <number>65536</number>
        </property>
        <property name="value">
         <number>8050</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Variable_Box</class>
   <extends>QLineEdit</extends>
   <header>nomad_camels.ui_widgets.variable_tool_tip_box</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEdit_x_axis</tabstop>
  <tabstop>lineEdit_nPoints</tabstop>
  <tabstop>lineEdit_title</tabstop>
  <tabstop>lineEdit_xlabel</tabstop>
  <tabstop>lineEdit_ylabel</tabstop>
  <tabstop>lineEdit_ylabel2</tabstop>
  <tabstop>lineEdit_top_left_x</tabstop>
  <tabstop>lineEdit_top_left_y</tabstop>
  <tabstop>lineEdit_plot_width</tabstop>
  <tabstop>lineEdit_plot_height</tabstop>
  <tabstop>checkBox_xlog</tabstop>
  <tabstop>checkBox_ylog</tabstop>
  <tabstop>checkBox_ylog2</tabstop>
  <tabstop>checkBox_same_fit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
